### PR TITLE
feat(languageSettings): add Markdown settings

### PR DESCRIPTION
**Pull request by kenji-miyake**
Addresses this error: https://github.com/autowarefoundation/autoware.universe/pull/426/files#diff-de1a19dc7002bcb047e7cd1b40c2e0b7ad0f7d5cd07d936df328c7532f4fabbaR28

image

Related: https://github.com/streetsidesoftware/vscode-spell-checker/issues/242#issuecomment-832931390

### CODE REVIEW

**Comment by kenji-miyake** [comment]
Or ignoring code sections is better?
https://github.com/streetsidesoftware/vscode-spell-checker/issues/242#issuecomment-406789352

File: .cspell.json [hunk 1]
```diff
@@ -40,6 +40,16 @@
     "\\[.*/.*\\]\\(https://github.com",
     "github.com[/:][\\w._\\-]+(/[\\w._\\-]+)?"
   ],
+  "languageSettings": [
+    {
+      "languageId": "markdown",
+      "dictionaries": [
+        "cpp",
+        "latex",
```
Comment (KeisukeShima) [inline .cspell.json:48]: Why did you add latex? I think it's not used in autoware.

**Review by KeisukeShima — COMMENTED** [review]
LGTM. Added a small question.

File: .cspell.json (hunk 1 above)
Comment (kenji-miyake) [reply .cspell.json:48]: It's used in Planning/Control documents.
https://github.com/autowarefoundation/autoware.universe/blob/4536e52bdbc80ea3dec0d2940ded51ffd118b477/planning/obstacle_avoidance_planner/obstacle_avoidance_planner-design.ja.md

Comment (KeisukeShima) [reply .cspell.json:48]: I see. Thanks!

**Review by KeisukeShima — APPROVED** [review]

**Comment by kenji-miyake** [comment]
I'll test https://github.com/tier4/autoware-spell-check-dict/pull/265#issuecomment-1099485140 later and merge this.

**Comment by kenji-miyake** [comment]
@KeisukeShima I've replaced the approach with ignoring code sections, and rebased on https://github.com/tier4/autoware-spell-check-dict/pull/268.

**Review by KeisukeShima — APPROVED** [review]
